bindings/python/unicorn/unicorn.py

```diff
@@ -9,7 +9,7 @@
 import sys
 import weakref
 
-from . import x86_const, unicorn_const as uc
+from . import x86_const, arm64_const, unicorn_const as uc
```

Comment:
are you sure this is necessary, as arm64_const is already imported in __init__.py?